wiki/Tournaments/JFC/1/en.md

```diff
@@ -231,6 +241,20 @@ The Jogja Fun Cup is run by various osu! community members from Indonesia and be
 
 ## Match results
 
+### Finals (week 2)
+
+Saturday, 10 August 2024:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | **Bakpia Kukus** | **7** | 1 | Ban HR2 | [#1](https://osu.ppy.sh/community/matches/115022242) |
+
+Sunday, 11 August 2024, Grand Final:
```

Comment:
this isn't marked elsewhere (it's the last match anyway)

```suggestion
Sunday, 11 August 2024:
```

Reply:
e i've been doing this everywhere else for years tho D:

![image](https://github.com/user-attachments/assets/288e181b-97a7-45db-b151-4a33557dd946)

![image](https://github.com/user-attachments/assets/0658b1d3-5542-4fef-b3e4-29a4e8adb975)

Reply:
i think TicClick suggested this to me once and i've been adding this distinction ever since

Reply:
ah, alright then